src/applications/ivc-champva/10-7959a/chapters/claimInformation.js

```diff
@@ -197,8 +195,8 @@ export const eobUploadSchema = isPrimary => {
           }`;
         },
         ({ formData }) => {
-          const name = nameWording(formData, true, true, true);
-          const yourOrTheir = name === 'your' ? name : 'their';
+          const name = nameWording(formData, true, false, true);
+          const yourOrTheir = name.toLowerCase() === 'your' ? name : 'their';
```

Comment:
adjusted `nameWording` params so it returns a lowercased "your", and updated `yourOrTheir` check to be case-insensitive as an extra precaution.